.github/ensure-tester/action.yml

```diff
@@ -70,7 +70,10 @@ runs:
 
     - name: Print Startup Log
       shell: bash
-      run: cat /run/log.out || true # don't fail
+      run:
+        # Try 6 times to see if .user-data-finished has been written to
+        for i in {0..6} ; do scripts/run_on_tester "cat /home/ubuntu/.user-data-finished" && break ; sleep 5 ; done ;
+        scripts/run_on_tester "cat /home/ubuntu/.user-data-finished ; cat /run/log.out" || true # don't fail
```

Comment:
Nit: I don't know off the top of my head whether `;` or `||` takes precedence in bash, so I don't know what's allowed to fail here.

Reply:
Just removed the || true - we should find the run/log.out